src/DqtApi/V2/Requests/GetTrnDetailsRequest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DqtApi.V2.Responses;
+using MediatR;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace DqtApi.V2.Requests
+{
+    public class GetTrnDetailsRequest : IRequest<GetTrnDetailsResponse>
+    {
```

Comment:
Would be good to have a `[FromQuery(Name = "camelCasedName")]` on each of these properties  (like `GetTrnRequest`) so we have correct naming style in the Swagger doc